core/services/pipeline/orm.go

```diff
@@ -410,6 +413,17 @@ WHERE pipeline_runs.id = batched_pipeline_runs.id`,
 		return errors.Wrap(err, "DeleteRunsOlderThan failed")
 	}
 
+	mark := time.Now()
+	o.lggr.Debugw("pipeline_runs reaper DELETE query completed", "duration", mark.Sub(start))
+
+	err = q.ExecQ("VACUUM ANALYZE pipeline_runs")
+	if err != nil {
+		o.lggr.Warnw("DeleteRunsOlderThan successfully deleted old pipeline_runs rows, but failed to run VACUUM ANALYZE", "err", err.Error())
+		err = nil
```

Comment:
Can we return nil instead?
If this was meant to route us back to always log the elapsed duration at the end, then maybe a defer could be registered instead?